app.js

```diff
@@ -1,3 +1,5 @@
+/* Hometask1 */
+
 let usersList = [];

```

Comment:
global variables shouldn't be used for data, pass it as a property to component where it should be used

Reply:
Done